app/models/user.rb

```diff
@@ -1,2 +1,13 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  # Method to return the 3 most recent posts for a given user.
+  def recent_posts
+    Post
+      .where(author_id: id)
+      .order(updated_at: :desc)
+      .limit(3)
+  end
```

Comment:
- [ OPTIONAL ] I think that you should simply retrieve a user posts by calling the `order()` method on the `posts` object and sorting them by the `created_at` attribute.
  
  ```ruby
  def recent_posts
    posts.order('created_at Desc').limit(3)
  end
  ```